include/deal.II/grid/grid_tools.h

```diff
@@ -1285,6 +1296,93 @@ namespace GridTools
       const bool                                             perform_handshake,
       const bool enforce_unique_mapping = false);
 
+
+    /**
+     * Data structure returned by
+     * GridTools::internal::distributed_compute_intersection_locations(). It can
+     * be converted to
+     * GridTools::internal::DistributedComputePointLocationsInternal, which can
+     * be used to reinit Utilities::MPI::RemotePointEvaluation.
+     */
+    template <int dim, int spacedim>
+    struct DistributedComputeIntersectionLocationsInternal
+    {
+      /**
+       * Intersection answers are assumed to be simplices (e.g. provided by
+       * CGAL)
+       */
+      using IntersectionAnswerType =
```

Comment:
```suggestion
      using IntersectionType =
```
... since `Answer` is not clear here.

Reply:
[6b74a7c](https://github.com/dealii/dealii/pull/15165/commits/6b74a7c9142c13b4883a9a3aa67bc008e4fa39db)